yapapi/executor/ctx.py

```diff
@@ -239,8 +239,8 @@ def run(
         :param env: optional dictionary with environmental variables
         :return: None
         """
-        stdout = CaptureContext.build()
-        stderr = CaptureContext.build()
+        stdout = CaptureContext.build(mode="stream")
+        stderr = CaptureContext.build(mode="stream")
```

Comment:
This will cause the output from the remote command to be streamed in chunks rather than in a single block.